lms/templates/student_account/login.underscore

```diff
@@ -1,7 +1,10 @@
 <div class="status already-authenticated-msg hidden">
     <% if (context.currentProvider) { %>
         <p class="message-copy">
-            <%- _.sprintf( gettext("You have successfully signed into %(currentProvider)s, but your %(currentProvider)s account does not have a linked %(platformName)s account. To link your accounts, Login now using your %(platformName)s password."), context ) %>
+            <%- _.sprintf( gettext("The %(currentProvider)s account you used is not associated to any Big Data University account."), context ) %>
```

Comment:
Don't hardcode `Big Data University` in the platform code. Use `%(platformName)s`